### PR TITLE
Standardize bridge API to improve typing

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch7052]

File: src/api/api.ts [hunk 1]
```diff
@@ -19,8 +20,7 @@ const requestSigning = (
   return new Promise((resolve, _reject) => {
     webContents.send(IPC_CHANNELS.API, request);
 
-    ipcMain.on(IPC_CHANNELS.API, function _listener(_event, arg) {
-      const response = arg as Omit<JsonRPCResponse, 'jsonrpc'>;
+    ipcBridgeMain(ipcMain).api.on(function _listener(_event, response) {
```
Comment (Mrtenz) [inline src/api/api.ts:23]: Can we use an arrow function here, and remove the `_`?

**Review by Mrtenz — CHANGES_REQUESTED** [review]
Would be cool if we can fully type JSON RPC requests and responses as well, but that may be out of scope for this PR.

File: src/api/api.ts (hunk 1 above)
Comment (FrederikBolding) [reply src/api/api.ts:23]: It has to be a named function as otherwise we can't remove the listener later. But I can remove the `_`

**Comment by FrederikBolding** [comment]
> Would be cool if we can fully type JSON RPC requests and responses as well, but that may be out of scope for this PR.

Agreed!

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**